src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.Windows.cs

```diff
@@ -127,6 +127,34 @@ public SocketInformation DuplicateAndClose(int targetProcessId)
             return info;
         }
 
+        public IAsyncResult BeginAccept(int receiveSize, AsyncCallback? callback, object? state)
+        {
+            return BeginAccept(null, receiveSize, callback, state);
```

Comment:
I know you're just moving it to a new file, but while we're here...

```suggestion
            return BeginAccept(acceptSocket: null, receiveSize, callback, state);
```